app/templates/base.html

```diff
@@ -79,7 +83,18 @@
     
 </div>
 
+<<<<<<< HEAD
+<!---- footer--> 
+<div class="footer">
+    <link><a href="privacy_policy.html" > Privacy Policy</a></link>
+    <link><a href="about_us.html" > About Us</a></link>
+    <link><a href="blog.html" > Blog</a></link>  <br>
+    <link><a href="terms.html" > Terms</a></link>
```

Comment:
this doesnt match with the file name as the file name is aboutus.html and then in the the base file its been linked to a about_us.html file? naming conventions have to be the same right